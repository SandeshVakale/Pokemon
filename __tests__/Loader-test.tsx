import 'react-native';
import React from 'react';
import Loader from '../src/components/loader';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders loader correctly', async () => {
  const tree = renderer.create(<Loader />).toJSON();

  expect(tree).toMatchSnapshot();
});
