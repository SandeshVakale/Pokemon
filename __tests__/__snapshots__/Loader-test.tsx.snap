// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loader correctly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator />
</RCTSafeAreaView>
`;
